test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildACppCliProjectWithTransitiveDeps.cs

```diff
@@ -12,7 +12,7 @@ public GivenThatWeWantToBuildACppCliProjectWithTransitiveDeps(ITestOutputHelper
 
         private readonly Lazy<TestAsset> _buildAsset;
 
-        [FullMSBuildOnlyFact(Skip = "https://github.com/dotnet/sdk/issues/3785")]
+        [FullMSBuildOnlyFact]
```

Comment:
Unskipping this test failed. Marking it with the successor issue. Details here: https://github.com/dotnet/sdk/issues/11008#issuecomment-2183528729